capstone/capdb/migrations/0026_auto_20180227_1659.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2018-02-27 16:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.utils.text import slugify
+from scripts.helpers import chunked_iterator
+
+
+def create_normalized_cite(apps, schema_editor):
```

Comment:
I don't think it's worth running these python updates, since we have to reindex all the cases anyway to rebuild the relationships. I think just wipe the data from the citation table before we deploy this.